File: AULA4/app/src/main/java/com/example/aula04/MainActivity.java
package com.example.aula04;

import android.os.Bundle;
import android.widget.Button;
import android.widget.TextView;

import androidx.activity.EdgeToEdge;
import androidx.appcompat.app.AppCompatActivity;
import androidx.core.graphics.Insets;
import androidx.core.view.ViewCompat;
import androidx.core.view.WindowInsetsCompat;

public class MainActivity extends AppCompatActivity {

    livro Livro1, Livro2, Livro3;

    TextView textInfo;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        EdgeToEdge.enable(this);
        setContentView(R.layout.activity_main);

        Livro1 = new livro("Primeiro Livro", "Primeiro Autor", 2026);
        Livro2 = new livro("Segundo Livro", "Segundo Autor", 2020);
        Livro3 = new livro("Terceiro livro", "Terceiro Autor", 2020);

       textInfo = findViewById(R.id.textInfo);

       Button button1 = findViewById(R.id.button);
       Button button2 = findViewById(R.id.button2);
       Button button3 = findViewById(R.id.button3);

       button1.setOnClickListener(  v -> textInfo.setText(Livro1.getInfo()));
       button2.setOnClickListener(  v -> textInfo.setText(Livro2.getInfo()));
       button3.setOnClickListener( v -> textInfo.setText(Livro3.getInfo()));

    }
    }

File: Projeto_aula4/app/src/main/java/com/example/projeto_aula4/MainActivity.kt
package com.example.projeto_aula4;

import android.os.Bundle;
import android.view.View
import  android.widget.Button;

import androidx.activity.enableEdgeToEdge;
import androidx.appcompat.app.AppCompatActivity;
import androidx.core.view.ViewCompat;
import androidx.core.view.WindowInsetsCompat;



public class MainActivity : AppCompatActivity() {

    Livro livro1, livro2, livro3;

    TextView textInfo
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        enableEdgeToEdge()
        setContentView(R.layout.activity_main)

        livro1 =  new Livro("Primeiro livro", "primeiro autor", "2020" );
        livro2 =  new Livro("Segundo livro", "Segundo  autor", "2020" );
        livro3 =  new Livro("Terceiro livro", "terceiro autor", "2020" );

        textInfo = findViewById (R.id.textView)

        Button button1 = findViewById (R.id.button);
        Button button2 = findViewById (R.id.button2);
        Button button3 = findViewById (R.id.button3);

        button.setOnClickListener (View v -> textInfo.setText(livro1.setInfo()) );
        button2.setOnClickListener (View v -> textInfo.setText(livro2.setInfo()) );
        button3.setOnClickListener (View v -> textInfo.setText(livro3.setInfo()) );


        }

    }
